File: serverTest/serverTest/Camera.cs
using System.Net;
using System.Net.Http;
using System.Threading.Tasks;
using NUnit.Framework;

namespace serverTest {
    public class Camera : Base {

        
        [TestCase( ServerUrl + "gallery/camera")]
        public async Task IsAllCamerasAvailable(string route) {
            int camersCount = 5;
            for (int camera = 1; camera < camersCount; camera++) {
                HttpResponseMessage response = await Client.GetAsync($"{route}/{camera}");
                if (response.EnsureSuccessStatusCode().StatusCode != HttpStatusCode.OK) {
                    Assert.Warn($"Camera {camera} is not found");
                }
            }
        }
    }
}

File: tests/Camera.py
import unittest
from tests.Base import Base
import requests


class Camera(Base):
    routeUrl = Base.serverUrl + "gallery/camera"

    camersCount = 5
    def IsAllCamerasAvailable(self):
        for camera in range(self.camersCount):
            r = requests.get(f"{self.routeUrl}/{camera}")
            if r.status_code != 200:
                self.assertTrue()


if __name__ == '__main__':
    unittest.main()



